src/astro/sql/operators/drop.py

```diff
@@ -31,3 +31,17 @@ def execute(self, context: Dict) -> Table:  # skipcq: PYL-W0613
         self.table = database.populate_table_metadata(self.table)
         database.drop_table(self.table)
         return self.table
+
+
+def drop_table(
+    table: Table,
+    **kwargs: Any,
+):
+    """
+    Drops a table.
+
+    :param table: Table to be dropped
+    :param kwargs:
```

Comment:
```suggestion
```
Are those the BaseOperator ones? If yes, we don't necessary need to document them. If they contain other kwargs as well, let us document them :)

Reply:
Done in https://github.com/astronomer/astro-sdk/pull/637/commits/98fd8a561cb136b865525b33b5b1207dffd1d122